test/config_templates/mongo_init.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/python
+
+import pymongo
+
+MONGODB_URI = 'mongodb://172.31.128.121:27017/admin'
+
+
+def main():
+
+    client = pymongo.MongoClient(MONGODB_URI)
+
+    config = {{mongo_list}}
```

Comment:
You can silence hound on a case-by-case basis by adding a comment at the end of the line like:
```    config = {{mongo_list}}  # NOQA: undefined name```